### DISCUSSION TITLE
Disable crops rotation on multiselection

### DISCUSSION THREAD

**Discussion by ZeroRetro** [berviantoleo/react-multi-crop#1845] [Ideas]
When I select a crop I can't rotate it. I would like to disable the rotation on multi-selection too. Also, I don't know why a single crop can't be rotated

https://github.com/berviantoleo/react-multi-crop/assets/33663801/3c0e433f-fbfa-43f0-835f-a0c9b13f8763

**Comment by berviantoleo** [comment]
Currently, the rotate function is disabled in this component. My previous client doesn't want to rotate the crop.

I might enable the feature, but it will be disabled by default.

**Reply by ZeroRetro** [reply]
I want to disable it too, but on multiselection you still can rotate the crops.

I think that having a flag to enable/disable will be useful, having them disabled by default its ok

**Reply by berviantoleo** [reply]
Ah, I will look at the multiselection.

**Reply by ZeroRetro** [reply]
Would be possible to remove the "corner" for rotation if its disabled?

![Screenshot 2023-05-17 at 11 03 05](https://github.com/berviantoleo/react-multi-crop/assets/33663801/f404d91c-1d58-4503-b6d2-642a9d183355)

**Reply by berviantoleo** [reply]
Possible

![image](https://github.com/berviantoleo/react-multi-crop/assets/15927349/d27239d7-ed19-484c-8361-57ab6bd2361a)

**Reply by berviantoleo** [reply]
Weirdly, I don't get any object to customize the multi-selection.

But, I can implement your query about the control for single selection.

> Would be possible to remove the "corner" for rotation if its disabled?

**Reply by ZeroRetro** [reply]
If isn't possible to disable the rotation in multi-selection what about disable the multi-selection?

**Reply by berviantoleo** [reply]
> If isn't possible to disable the rotation in multi-selection what about disable the multi-selection?

Possible to disable the multi-selection. I will try to expose those props and set disabled as default. What do you think?

**Reply by ZeroRetro** [reply]
Yes, I think that will be useful, thanks!

**Reply by berviantoleo** [reply]
I found a bug that the canvas doesn't want to disable the multi-selection. It might take a long time to debug. 😄

**Reply by berviantoleo** [reply]
> Would be possible to remove the "corner" for rotation if its disabled?

I'll release this first when I'm debugging to handle the multi-selection.
